src/test/scala/microsites/LayoutTest.scala

```diff
@@ -79,6 +79,32 @@ class LayoutTest extends FunSuite with Checkers with Matchers with Arbitraries {
     check(property)
   }
 
+  test("globalFooter TypeTag should have the appropriate content") {
+
+    val property = forAll { implicit settings: MicrositeSettings ⇒
+      (settings.templateTexts.footer.isDefined) ==> {
```

Comment:
Is there a ligature for `==>`?

Reply:
Not that I'm aware of but AFAIK we don't use special symbols anywhere since many fonts already display ligatures for those

Reply:
Only reason I ask is this project has `⇒` explicitly on Line 84 and 47deg/fetch has scalafmt rewrite rules to remove them(https://github.com/47deg/fetch/blob/master/.scalafmt.conf#L24). I was curious if we are requiring/maintaining consistency.